kvs-rest-clients/src/main/java/org/gbif/rest/client/grscicoll/GrscicollLookupResponse.java

```diff
@@ -48,4 +44,32 @@ public static class EntityMatchedResponse implements Serializable {
     private UUID key;
   }
 
+  public enum MatchType {
```

Comment:
can't we use the enums of the gbif-api?

Reply:
again, im trying to reduce dependencies where possible. There isnt any real value add from using the enum.

Reply:
ok, let's leave as it is then. The only downside is to remember to update them if they change but they rarely change now.